content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -238,55 +219,44 @@ Now you see, we’ve created a new dog with the name Bingo. You can go on to cre
  
 Here is the output:
  
-    Dog running
-    
-    Is the dog running:? true
-    
-    Sleeping
-    
-    is the dog sleeping:? true
-    
-    Dog barking...
-    
-    Is the dog barking:? true
-    
-    Eating
-    
-    Is the dog eating:? true
-    
-    The dog name is: Bingo
-    
-Now that we’ve got this out of the way let's talk about abstraction and encapsulation.
+  Dog running
  
-### Abstraction
+  Is the dog running:? true
  
-Who could have imagined that abstraction will be a thing in programming? Who? But right now, you can’t talk about OOP without mentioning abstraction. It is a significant building block of OOP. So then, what is abstraction? This section will explain what abstraction is and how to use it as a Java developer.
+  Sleeping
  
-In abstraction, only the useful attributes of an object are on display or accessible. The unnecessary details are “hidden”.
+  is the dog sleeping:? true
  
-For example, while you’re typing on your computer. You’re interested in punching the keys believing that it produces results on your screen. You know nothing about the mechanism that produces the result, and I’m sure you haven’t even thought about it yet. Have you?
+  Dog barking...
  
-Here's another example to make things clearer: imagine a car driver. He is only interested in starting, moving, and stopping the car, speed, or change gear and brake, but he is not interested in the mechanism that allows him to do this. In abstraction, we hide the implementation details from the user and expose the functionality.
+  Is the dog barking:? true
  
-This way, the user will only know what a function(method in java) does but not how it does it.
+  Eating
+ 
+  Is the dog eating:? true
  
-### How to implement Abstraction in Java:
+  The dog name is: Bingo
+ 
+Now that we’ve got this out of the way let's talk about abstraction and encapsulation.
+ 
+### Abstraction 
+Who could have imagined that abstraction will be a thing in programming? Who? But right now, you can’t talk about OOP without mentioning abstraction. It is a significant building block of OOP. So then, what is abstraction? This section will explain what abstraction is and how to use it as a Java developer.
+
+In abstraction, only the useful attributes of an object are on display or accessible. The low-level details are “hidden”. For example, while you’re typing on your computer. You’re interested in punching the keys believing that it produces results on your screen. You know nothing about the mechanism that produces the result, and I’m sure you haven’t even thought about it yet. Have you?
```

Comment:
>"hidden"

replace the quotation marks with asterisks (\*) to italicize it.

Reply:
>For example, while you’re typing on your computer. You’re interested in punching the keys believing that it produces results on your screen.

Replace the period between these sentences with a comma and make *You're* lower case:

> For example, while you’re typing on your computer, you’re interested in punching the keys believing that it produces results on your screen.